include/envoy/ssl/connection.h

```diff
@@ -25,6 +25,12 @@ class Connection {
    **/
   virtual std::string uriSanLocalCertificate() PURE;
 
+  /**
+   * @return the subject field of the local certificate in RFC 2253 format. Returns "" if there is
```

Comment:
+1 on citing the RFC in the comment.  Can you please also put a simple example value, so I don't have to look up the RFC to know approximately what this function returns?

Reply:
@saumoh Can you either make the doc change here or comment on why it's not a good idea?

Reply:
@ggreenway I should have mentioned that i added some sample values in one of the follow up  commits.
https://github.com/envoyproxy/envoy/pull/2319/commits/409f159227bbf6d78516afec480914cb32d148f2#diff-ded6386c66f63a1375432303053ca7eeR290